### PR TITLE
[CMLG- 019] Implement the basic structure of header bar

### PR DESCRIPTION
**Pull request by EileenSiqi**
Issue:
The user wants to toggle between different pages, so we need a header bar.

Solution: 
Add a headerBar component and use Route to switch between different components.

Risk:
CSS still needs to be modified.

Reviewed by:
Kevin, Emily, Yujia, Annie, Dave

### CODE REVIEW

**Review by ywu666 — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -15,10 +17,22 @@ export default function App() {
             renders the first one that matches the current URL. */}
         <Switch>
           <Route path="/translations">
-            <SearchPage />
+            <div>
+              <HeaderBar />
```
Comment (annithinggoes) [inline src/App.js:21]: Not entirely sure how, but could you consider a way to move the header bar component to be outside the switch somehow? so regardless of the route the header would be there?

File: src/components/AboutUs.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+class AboutUs extends React.Component {
+    render() {
```
Comment (annithinggoes) [inline src/components/AboutUs.js:4]: Would name it as AboutUsPage to be consistent with SearchPage and WelcomePage

**Review by annithinggoes — APPROVED** [review]
Small details, for the actual svg no idea whats going on so I can't check it.

File: src/App.js (hunk 1 above)
Comment (EileenSiqi) [reply src/App.js]: Good point! I've changed the code so that the header bar component is outside the switch now.

**Review by zyan225 — APPROVED** [review]
Looks good, could consider using Font Awesome instead of svg.
I don't know if there is any difference between the functionality of two, but Font Awesome will make the code much clearer.

**Comment by EileenSiqi** [comment]
Done! I've changed to the font awesome icons.

**Pull request merged**